source/src/microapp/cs_MicroappController.cpp

```diff
@@ -526,6 +550,21 @@ bool MicroappController::allowSoftInterrupts(MicroappSdkType type, uint8_t id) {
 	return true;
 }
 
+bool MicroappController::isCpuBusy() {
+	uint16_t schedulerSpace = app_sched_queue_space_get();
+	if (schedulerSpace < SCHED_QUEUE_SIZE - SCHEDULER_QUEUE_ALMOST_FULL) {
```

Comment:
Shouldn't this just be `SCHEDULER_QUEUE_ALMOST_FULL`?

```
#define SCHED_QUEUE_SIZE                         32
#define SCHEDULER_QUEUE_ALMOST_FULL              (SCHED_QUEUE_SIZE - 10)
```

So `SCHED_QUEUE_SIZE - SCHEDULER_QUEUE_ALMOST_FULL == 10.`

Reply:
When the queue is almost full, there are 22 items queued. So there is space for 10.